app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,43 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+  <%= f.label :name %><br />
+  <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :bio, class: "form-label" %>
+    <%= f.text_field :bio, autofocus: true, autocomplete: "bio", placeholder: "Bio", class: "custom-input", required: true %>
+  </div>
+
+  <div class="field">
+    <%= f.hidden_field :photo, value: "https://previews.123rf.com/images/afe207/afe2071811/afe207181100114/112623425-imagen-de-perfil-masculino-s%C3%ADmbolo-de-icono-de-avatar-de-perfil-de-silueta-con-gafas.jpg" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
```

Comment:
- [ ] @Team Great work adding the registration page, however, the layout doesn't match the design from the wireframe given , check the comparison below:


Given Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/GhausAliKhan/Blog-app/assets/61408860/5c22d079-7996-48d7-84e4-119e82e1eb5b) |  ![image](https://github.com/GhausAliKhan/Blog-app/assets/61408860/ab902e9d-f59e-490a-bbc9-204f5d734f8e)